unleashandroidsdk/build.gradle.kts

```diff
@@ -77,6 +77,7 @@ dependencies {
     testImplementation(libs.robolectric.test)
     testImplementation(libs.okhttp.mockserver)
     testImplementation(libs.awaitility)
+    testImplementation(libs.jsonunit)
```

Comment:
This comes with SLF4J but it's in the test scope

Reply:
Should be fine